lib/pronto/config_file.rb

```diff
@@ -35,8 +36,12 @@ class ConfigFile
       'format' => DEFAULT_MESSAGE_FORMAT
     }.freeze
 
-    def initialize(path = '.pronto.yml')
-      @path = path
+    def initialize
+      @path = if File.exist?(ENV['PRONTO_CONFIG'].to_s)
```

Comment:
Because of https://github.com/mmozuras/pronto/pull/228/files#diff-d06fbb141ccb6c55a35f4ed5668f6752R19 `PRONTO_CONFIG` will always have a default value. That mens that if no config file exists (which isn't unusual) it would raise an error. But I agree that if it's set to a different value and the file does not exist that it should raise an error.

I didn't think about `Hash#fetch`. I'll update the code. 👍